a/asusd/stone.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : asusd
+version     : 6.1.0-rc2
+release     : 1
+summary     : Control ASUS laptop features easily
+license     :
+    - MPL-2.0
+homepage    : https://asus-linux.org/
+upstreams   :
+    - git|https://gitlab.com/asus-linux/asusctl.git : d785e17f9518c414134ce00585f1d83c0698090c
+description : |
+    Control ASUS platform features such as LEDs, AniMe, fans and more.
+builddeps   :
+    - binary(cargo)
+    - binary(cc)
+    - binary(clang)
+    - binary(make)
+    - pkgconfig(libinput)
+    - pkgconfig(libudev)
+    - pkgconfig(libseat)
+    - pkgconfig(libzstd)
+    - pkgconfig(libpcre2-8)
+    - pkgconfig(expat)
+    - pkgconfig(gbm)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(xkbcommon)
```

Comment:
I feel like many of these pkgconfig dependencies are unnecessary and can be removed. I took a look at the project source code and (correct me if I'm wrong since you seem heavily involved in the project) it looks like it's only or mostly only Rust code. For rust code you can generally infer system library dependencies by checking the Cargo.lock and looking for `-sys` depenencies and dependencies that have `cc` or `pkg-config` as dependencies of themselves. While I see a few such dependencies many of them don't seem to correspond to the dependencies in the list here, nor are they getting linked against (see the `soname(*)` dependencies in `manifest.jsonc`).

Reply:
the crate `slint` requires most of these.